javaparser-core/src/main/java/com/github/javaparser/ast/visitor/ModifierVisitor.java

```diff
@@ -1097,5 +1092,79 @@ public Visitable visit(final LineComment n, final A arg) {
     private <N extends Node> NodeList<N> modifyList(Optional<NodeList<N>> list, A arg) {
         return list.map( ns -> modifyList(ns, arg)).orElse(null);
     }
+
+    public Visitable visit(ModuleDeclaration n, A arg) {
+        NodeList<AnnotationExpr> annotations = modifyList(n.getAnnotations(), arg);
+        NodeList<ModuleStmt> moduleStmts = modifyList(n.getModuleStmts(), arg);
+        Name name = (Name) n.getName().accept(this, arg);
+        Comment comment = n.getComment().map( s -> (Comment) s.accept(this, arg)).orElse(null);
+        if (name == null)
+            return null;
+        n.setAnnotations(annotations);
+        n.setModuleStmts(moduleStmts);
+        n.setName(name);
+        n.setComment(comment);
+        return n;
+    }
+
+    public Visitable visit(ModuleRequiresStmt n, A arg) {
+        Name name = (Name) n.getName().accept(this, arg);
+        Comment comment = n.getComment().map( s -> (Comment) s.accept(this, arg)).orElse(null);
+        if (name == null)
+            return null;
+        n.setName(name);
+        n.setComment(comment);
+        return n;
+    }
+
+    @Override()
+    public Visitable visit(ModuleExportsStmt n, A arg) {
+        NodeList<Name> moduleNames = modifyList(n.getModuleNames(), arg);
+        Name name = (Name) n.getName().accept(this, arg);
+        Comment comment = n.getComment().map( s -> (Comment) s.accept(this, arg)).orElse(null);
+        if (name == null)
+            return null;
+        n.setModuleNames(moduleNames);
+        n.setName(name);
+        n.setComment(comment);
+        return n;
+    }
+
+    @Override()
+    public Visitable visit(ModuleProvidesStmt n, A arg) {
+        Type type = (Type) n.getType().accept(this, arg);
+        NodeList<Type> withTypes = modifyList(n.getWithTypes(), arg);
+        Comment comment = n.getComment().map( s -> (Comment) s.accept(this, arg)).orElse(null);
+        if (type == null)
+            return null;
+        n.setType(type);
+        n.setWithTypes(withTypes);
+        n.setComment(comment);
+        return n;
+    }
+
+    @Override()
+    public Visitable visit(ModuleUsesStmt n, A arg) {
+        Type type = (Type) n.getType().accept(this, arg);
+        Comment comment = n.getComment().map( s -> (Comment) s.accept(this, arg)).orElse(null);
+        if (type == null)
```

Comment:
Actually, I have a better answer. The code in the generator is a little more readable because it doesn't contain the braces :)